1-mandatory/1-writers.js

```diff
@@ -52,6 +52,16 @@ Exercise 1:
 */
 function logAllWriters() {
   // write your code to log all writers here
+/*   for (let i = 0; i < arr.length; i++) {
+    console.log(
+      `Hi, my name is ${arr[i].firstName} ${arr[i].lastName}. I am ${arr[i].age} years old, and work as a writer.`
+    );
+  } */
+  for (let i = 0; i < writers.length; i++) {
+    console.log(
+      `Hi, my name is ${writers[i].firstName} ${writers[i].lastName}. I am ${writers[i].age} years old, and work as a writer.`
+    );
+  }
```

Comment:
This works fine, but `.forEach()` would be more preferred whenever you need iterate over every element of the array.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach